client/src/components/EventCard/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Card, CardActionArea, CardContent} from "@mui/material";
+import ProgrammingLanguages from "./ProgrammingLanguages";
+import Buttons from "./Buttons";
+import HeadingDatesAndIcons from "./HeadingDatesAndIcons";
+import SponsoredBy from "./SponsoredBy";
+
+const EventCard = () => {
+    return (
+        <Card
+            sx={{
+                width: "80%",
+                borderRadius: "10px",
+                margin: "auto",
+                marginTop: "5%",
+                boxShadow: "0px 0px 1.5px 0px #fff",
```

Comment:
https://mui.com/system/shadows/#api

Reply:
Made a custom `boxShadow` inside the theme, under `shadows[]`.